tests/Samples.Server.Tests/Serializer.cs

```diff
@@ -0,0 +1,18 @@
+#if NETCOREAPP2_2
+using Newtonsoft.Json;
+#else
+using System.Text.Json;
+#endif
+
+namespace Samples.Server.Tests
+{
+    public static class Serializer
```

Comment:
This code file can be merged with one from _src/Ui.Playground/Shared/Serializer.cs_ to be in one place but I think it will not bring much benefit and you can leave everything as it is with a little of copy-paste.

Reply:
I tried to do this, but the Samples.Tests.Serializer project relies on Ui.Playground and Ui.Altair, which both had this (Playground has the original, Altair a link), so using it there I couldn't figure out a way to tell the compiler which one to use. Linked files get the same file (including the namespace). 

Gave up and just copy-pasted. Got any other suggestions?

Reply:
Copy-paste is ok, it is a very small utility code.